pkg/cmd/repo/clone/clone.go

```diff
@@ -67,6 +71,8 @@ func NewCmdClone(f *cmdutil.Factory, runF func(*CloneOptions) error) *cobra.Comm
 	}
 
 	cmd.Flags().StringVarP(&opts.UpstreamName, "upstream-remote-name", "u", "upstream", "Upstream remote name when cloning a fork")
+	cmd.Flags().IntVarP(&opts.RetryLimit, "retries", "r", 3, "Maximum number of attempts")
+	cmd.Flags().IntVarP(&opts.RetryBackoff, "backoff", "b", 100, "Backoff interval")
```

Comment:
Let's not expose these details to our users. We can just use the default retries and backoff in all cases.